platform/android/src/ch/opengis/qfield/QFieldUtils.java

```diff
@@ -42,6 +42,19 @@
 
 public class QFieldUtils {
 
+    public static String replaceFilenameTags(String string, String filename) {
```

Comment:
Docstring ? 🙏

Reply:
@m-kuhn , we've never done documentation of the Java QFieldUtils class. Happy to start it, what's the syntax here?

Reply:
Javadoc, I'd propose.

Reply:
@m-kuhn , done.